app/helpers/activities_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ActivitiesHelper
+  def users_for_select
+    current_project.activities.map(&:user).uniq.union(current_project.authors.enabled)
```

Comment:
do we need a helper just for this?

Reply:
I think it's much more tidy to add it there. Wouldn't it be a lot of logic for a view?

Reply:
>Wouldn't it be a lot of logic for a view?

yes for a view it would be too much, but I was thinking maybe just have something like:
```ruby
@users_for_select = current_project.activities.map(&:user).uniq.union(current_project.authors.enabled)
```
In the controller's `index` action. But I'll leave this up to you and @aapomm

Reply:
This is mostly done in the controller unless we use it for other controllers or actions.
Also, what's the reasoning behind showing all the `current_project.authors` ? I imagine a user would only want to filter users that have activities, so ` current_project.activities.map(&:user)` should be enough?

Reply:
> Also, what's the reasoning behind showing all the current_project.authors ? I imagine a user would only want to filter users that have activities

If there are many activities by many users, an admin may not know if a particular user on the project has an activity or not (at least not at a glance). If the list of filterable users doesn't include a user who's on the project, it can lead to confusion: "Why can't I filter Aaron's activities if Aaron is on the project?" With this implementation, the admin will get a clear answer showing them that there are no activities that match the query.

Reply:
Removed the `ActivitiesHelper` and moved `@users_for_selec`t in the controller.